tests/unit/array/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED: 'true'
       PERSISTENCE_DATA_PATH: '/var/lib/weaviate'
   qdrant:
-    image: qdrant/qdrant:v0.7.0
+    image: qdrant/qdrant:v0.10.0
```

Comment:
please update to 0.8, not 0.10. 0.10 has spme other problems that will be dealt with in different PR